.gitignore

```diff
@@ -1,2 +1,41 @@
 .DS_Store
 node_modules
+
+*~
+.fuse_hidden*
+.directory
+.Trash-*
+.nfs*
+*.DS_Store
+.AppleDouble
+.LSOverride
+Icon
+._*
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pids
+*.pid
+*.seed
+*.pid.lock
+node_modules/
+jspm_packages/
+.npm
+.eslintcache
+*.tgz
+.env
+.next
```

Comment:
too big list, is that really necessary?

Reply:
This is the usual list I use for all common temporary file types.

`.eslintcache` is the essential addition of course